advanced/path_walker/path_walker.py

```diff
@@ -0,0 +1,125 @@
+import os
+
+ROOT_PATH = r"C:\Users\SNIR-PC\Desktop\backup\snir\school&courses\python_playground\kumpus_course\6-Modules"
+
+
+def recourse_files(root_path):
```

Comment:
Implement this function as a method of PathWalker.
if the object points at file, he just prints the file,
if it points to a directory, it uses the iteration (using your __iter__) 
to iterate over all his children PathWalker recurse_files method